doc/../device_guide/nrf54l: added KMU basics info page (#19407)
nrfconnect/sdk-nrf: doc/nrf/app_dev/device_guides/nrf54l/kmu_provision.rst

* Parameter ``--dev-id`` specifies the interface serial number and should be used if multiple J-link interfaces are connected to the development machine.

* Parameter ``-p" (--policy)`` specifies policy applied to the given set of keys. Specifiers are:
   * ``lock-last``: (default) last key is uploaded as locked, others as revokable.
   * ``revokable``: each key can be revoked.

>    * ``lock``: all keys are permanent.

Reviewer: Btw. Should we use `revocable` or `revokable` in the script? (currently script actually uses `revokable` as a supported CLI parameter name)

>    * ``lock``: all keys are permanent.

Reviewer 2: That would be my unfortunate correction - wasn't aware. When referring to the actual option we should definitely use the name from the script (i.e., revokable)

* Parameter ``--dev-id`` specifies the interface serial number and should be used if multiple J-link interfaces are connected to the development machine.

* Parameter ``-p" (--policy)`` specifies the policy applied to the given set of keys. 
  You can apply the following options:
   * ``lock-last`` - Uploads the last key as locked,  while the preceding keys are revocable. 
     This option is set by default.

>    * ``revocable`` - Enables revocation for each key.

Reviewer: Script supports `revokable` as CLI parameter name. This needs to be aligned too.

>    * ``revocable`` - Enables revocation for each key.

Reviewer 2: ```suggestion
   * ``revokable`` - Enables revocation for each key.
```

* Parameter ``--dev-id`` specifies the interface serial number and should be used if multiple J-link interfaces are connected to the development machine.

* Parameter ``-p" (--policy)`` specifies the policy applied to the given set of keys.
  You can apply the following options:

>       * ``lock-last`` - Uploads the last key as locked,  while the preceding keys are revocable. This option is set by default.

Reviewer: ```suggestion
      * ``lock-last`` - Uploads the last key as locked, while the preceding keys are revocable. This option is set by default.
```

      * ``revokable`` - Enables revocation for each key.
      * ``lock`` - Sets all keys to be permanent.

The script generates the public key for each private key and uploads them to your device.
These public keys generate the verification keys for the application image, which are then used by MCUboot for validation.
The first key specified in the command is used for signing the application image.
Currently, the script supports only ED25519 Keys.

> For MCUboot notice that it uses one key by default and KMU support in its configuration need to be enabled, otherwise MCUboot fallbacks to compiled in key.

Reviewer: ```suggestion
For MCUboot notice that:

* it uses one key by default,
* KMU support in its configuration needs to be enabled (with ``SB_CONFIG_MCUBOOT_SIGNATURE_USING_KMU`` sysbuild Kconfig option), otherwise MCUboot fallbacks to compiled in key.

```

For provision one key to the board run the following command:

.. parsed-literal::
   :class: highlight

>     west ncs-provision upload -s nrf54l15 -k ed25519.pem -p lock

Reviewer: They key should be locked even without the `lock` parameter (`lock-last` policy is used by default)

Author: I was going to emphasize that provisioning single key should be locked. will remove.
